.github/workflows/phpcs.yml

```diff
@@ -0,0 +1,45 @@
+name: Drupal coding standards
+
+on:
+  # Run on all pushes and on all pull requests.
+  # Prevent the build from running when there are only irrelevant changes.
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  Verify:
+    name: "Verify Drupal conding standards"
+
+    runs-on: "ubuntu-latest"
+
+    env:
+      CODE_SNIFFER_PATH: "modules/custom/ themes/custom/"
+      CODE_SNIFFER_IGNORE: "*md,*.css,*.txt,*.info,*interactive_ui/react_slider/js*,*pl-sk/*,*pattern-lab/*,*_twig-components/*"
+      CODE_SNIFFER_EXTENSIONS: "php,inc,install,module,yaml,twig,theme"
```

Comment:
What about the `profile` file extension?
What about the scss, js? Is it also validated?

Reply:
@mariusz-andrzejewski-droptica Good point, i added profile extension to check.
About scss and js files, I think better solution would be [Stylelint](https://stylelint.io/) and [ESLint](https://eslint.org/) like in Drupal core